doc/source/getting_started/install.rst

```diff
@@ -277,11 +277,12 @@ Installable with ``pip install "pandas[excel]"``.
 ========================= ================== =============== =============================================================
 Dependency                Minimum Version    pip extra       Notes
 ========================= ================== =============== =============================================================
-xlrd                      2.0.1              excel           Reading Excel
-xlsxwriter                3.0.5              excel           Writing Excel
-openpyxl                  3.1.0              excel           Reading / writing for xlsx files
+xlrd                      2.0.1              excel           Reading / writing for xls files
```

Comment:
xlrd doesn't write in pandas

Reply:
but xlrd the dependency itself can write.
How should write note for this?

Reply:
`Reading xls files`

Reply:
ok